backend/recruitmentapp/apps/core/models.py

```diff
@@ -42,3 +42,21 @@ class Recruiter(models.Model):
 
     def __str__(self):
         return self.user.username
+
+
+class Availability(models.Model):
+    """The dates an applicant is available to work."""
+    applicant = models.ForeignKey(Applicant, on_delete=models.CASCADE, related_name="availabilities")
+    start = models.DateField(verbose_name="Start date", null=False, blank=False)
+    end = models.DateField(verbose_name="End date", null=True, blank=True)
+
+    def __str__(self):
+        return str(self.applicants) + self.start + "-" + self.end
```

Comment:
This becomes something like "username2018-02-20-2018-02-28". You may want to add some spaces.